doc/source/conf.py

```diff
@@ -238,6 +238,7 @@ def __init__(self, version: str):
         r"http://localhost[:/].*",  # Ignore localhost links
         r"^http:/$",  # Ignore incomplete links
         # 403 Client Error: Forbidden for url.
+        "https://madewithml.com/#course",
```

Comment:
why is this one added here?

Reply:
I suspect that the host server for `madewithml.com` somehow doesn't accept Sphinx user agent for linkcheck, thus the 403 error. I manually verify and the link seems to work, so I added this domain to the ignore list.